samples/NovelRT.DotNet.Sample/CMakeLists.txt

```diff
@@ -1,52 +1,53 @@
 find_package(Dotnet ${NOVELRT_DOTNET_VERSION} REQUIRED)
 
-set(DOTNET_SAMPLE_CSPROJ ${CMAKE_CURRENT_LIST_DIR}/NovelRT.DotNet.Sample.csproj)
+set(CSPROJ ${CMAKE_CURRENT_LIST_DIR}/NovelRT.DotNet.Sample.csproj)
 
-set(DOTNET_SAMPLE_SOURCES
+set(SOURCES
   NovelRT.DotNet.Sample.runtimeconfig.json
   Program.cs
 )
 
-set(DOTNET_SAMPLE_OPTIONS
+set(OPTIONS
   /nologo
-  /p:IntermediateOutputPath=${CMAKE_CURRENT_BINARY_DIR}${CMAKE_FILES_DIRECTORY}/Dotnet_Sample_Build.dir/
-  /p:MSBuildProjectExtensionsPath=${CMAKE_CURRENT_BINARY_DIR}${CMAKE_FILES_DIRECTORY}/Dotnet_Sample_Build.dir/
-  /p:OutDir=${CMAKE_CURRENT_BINARY_DIR}/
-  /p:PublishDir=${CMAKE_CURRENT_BINARY_DIR}/publish/
+  /p:IntermediateOutputPath=${CMAKE_CURRENT_BINARY_DIR}${CMAKE_FILES_DIRECTORY}/Dotnet_Sample_Build.obj/
```

Comment:
Explicitly calling out that I renamed this from `.dir` to `.obj` and make `${CMAKE_CURRENT_BINARY_DIR}` the publish folder.

The binary outputs are now going to a `.bin` folder in the cmake files directory.

Reply:
This was updated to be `.dir/bin` and `.dir/obj` in the latest iteration.